engines/query-sparql-file/package.json

```diff
@@ -54,7 +53,6 @@
     "build": "yarn run build:ts",
     "build:ts": "node \"../../node_modules/typescript/bin/tsc\"",
     "prepare": "comunica-compile-config config/config-default.json > engine-default.js && comunica-compile-config config/config-browser.json > engine-browser.js",
-    "browser": "yarn run prepare && \"../../node_modules/webpack/bin/webpack.js\" --config webpack.config.js --mode production",
```

Comment:
Why was this one removed?

Reply:
Because file-based querying is unlikely to work in the browser, so I thought building a browser bundle for it was kind of unnecessary. But maybe I am wrong.

Reply:
It most definitely works in the browser, for querying (only) Linked Data documents.

Reply:
Ah okay, I thought it was specifically for files, not online documents. The name was a bit confusing.